10x/README.md

```diff
@@ -1,14 +1,64 @@
 ## 10x Genomics Pipelines
 
-This is code for running 10x data on the HCA Green Box. There are three directories:
+This is code for running 10x data on the HCA Green Box. It perform the 
+following tasks: 
 
-<dl>
-  <dt>WDLs</dt>
-  <dd>The WDLs to run the pipelines along with example inputs.</dd>
+1. Extracts barcode information from read 1 and index read 1
+2. Aligns reads containing genomic information to a reference genome (using STAR)
+3. Tags aligned bam reads with barcode information extracted in step 1
+4. Identifies duplicate reads
+5. Quantifies molecule counts in each gene and cell
+6. Filters erroneous cell and molecular barcodes
 
-  <dt>docker-10x</dt>
-  <dd>Docker image used for running 10x WDLs with Cromwell.</dd>
+# Download
+Use git clone:
+```
+git clone git@github.com:HumanCellAtlas/skylab.git
+cd skylab/10x/wdls/
+```
 
-  <dt>martian\_cli</dt>
-  <dd>Command line wrapper around the 10x Martian workflow executor.</dd>
-</dl>
+# Requirements
+## Dockers
+- 10x clean docker: `marcusczi/cellranger_clean:latest`
+
+## Inputs
+- `read_paths`: path to a fastq folder containing all sequencing reads for the experiment
+- `sample_def`: json file containing run information (see sample_def.json for example):  # TODO I omitted some here, should I have?
+  - `sample_names`: name for experiment
+  - `fastq_mode`: type of bcl demultiplexing to be done on the data
+  - `read_paths`: location for fastq information #todo duplicate of above??
+- `reads_per_file`: size of chunks (in reads) to create when scattering across processors
+- `primers`: Illumina P5 and P7 sequencing adapters used
+- `subsample_rate`: if passed and < 1.0, downsample data to this fraction. 
+- `align`: The software to use to align reads
+- `reference_path`: location of cellranger reference 
+  (constructed with `skylab/10x/generate_reference_bundle.wdl`)
+
+## Runtime Requirements
+Memory: 50 GB  # todo is this right? 
+Processors: 1+ (supports multiprocessing)
+Disk Space: Input data size * 5 (normally approximately _ ) GB # todo marcus what's your estimate here? 
```

Comment:
Another one that varies by step. The maximum usage for a single step seems to be roughly (size of three fastqs) * 5 + size of extracted reference